tests/unittests/bases/test_ddp.py

```diff
@@ -241,11 +231,10 @@ def reload_state_dict(state_dict, expected_x, expected_c):
 @pytest.mark.skipif(sys.platform == "win32", reason="DDP not available on windows")
 def test_state_dict_is_synced(tmpdir):
     """Tests that metrics are synced while creating the state dict but restored after to continue accumulation."""
-    torch.multiprocessing.spawn(_test_state_dict_is_synced, args=(2, tmpdir), nprocs=2)
+    pytest.pool.map(partial(_test_state_dict_is_synced, tmpdir=tmpdir), range(NUM_PROCESSES))
```

Comment:
Is this faster?

Reply:
Yes, because it is reusing the same pool over and over again instead of initializing a new.
On average it takes 5-6 sec to initialize a pool. We currently do that around 100 times in all our testing, thus we should save around 500-600 secs.